cmd/image-builder/README.md

```diff
@@ -181,22 +171,22 @@ The authentication to the Signify API has been updated from using `role id/secre
 - **mTLS Authentication**: Image Builder now uses a client certificate/private key pair for authentication with the Signify API. These credentials are valid for 7 days, after which they must be rotated.
 - **Automated Rotation**: The certificate rotation must occur every 7 days. The new certificate/private key pair must be generated using the previous pair before they expire.
 
-The Signify API's structure has also been updated. For more information, see the official [Signify API Documentation](https://pages.github.tools.sap/Repository-Services/Signify/how_to/manage_signatures/).
+The Signify API's structure has also been updated. For more information, see the official [Signify API Documentation](https://pages.github.tools.sap/Signify/docs/how_to/manage_signatures/).
 
 > [!NOTE]
-> Images are only signed when built on `push`, `schedule`, and `workflow_dispatch` events. Pull request and merge queue images are not signed.
+> Images are only signed when built on **push**, **schedule**, and **workflow_dispatch** events. Pull request and merge queue images are not signed.<!--isn't it the same as in lines 169-170?-->
 
 ### Signify API Changes
 
-The JSON structure for signing has changed. See the new structure and examples in the [Signify API Documentation](https://pages.github.tools.sap/Repository-Services/Signify/how_to/manage_signatures/).
+The JSON structure for signing has changed. See the new structure and examples in the [Signify API Documentation](https://pages.github.tools.sap/Signify/docs/how_to/manage_signatures/).<!--Could we put it together with line 179? Plus there have been some changes in the original docu - the link's changed - is it still valid?-->
 
-## Environment File
+## Environment File <!--The info is scarce here. Does it mean that the user only needs that much info, or could we skip it and keep only the more extended section on variables in the other doc?-->
```

Comment:
Lets add this details how to pass information about location to the env file. This should be enough for users.
```
The `--env-file` specifies the path to the file with environment variables to be loaded in the build.
```